pkg/vm/create_evm.go

```diff
@@ -127,8 +130,13 @@ func CreateEVMGenesis(
 	}
 	if params.UsePoAValidatorManager {
 		validatormanager.AddTransparentProxyContractToAllocations(params.initialTokenAllocation, proxyOwner)
-		validatormanager.AddValidatorMessagesContractToAllocations(params.initialTokenAllocation)
-		validatormanager.AddPoAValidatorManagerContractToAllocations(params.initialTokenAllocation)
+		if useACP99 {
+			validatormanager.AddValidatorMessagesACP99ContractToAllocations(params.initialTokenAllocation)
```

Comment:
deployedValidatorMessagesACP99Bytecode has the same value as validator manager 1.0.0 btw

Reply:
so can just reuse AddValidatorMessagesContractToAllocations

Reply:
did not tested. can we make another PR for this?